.github/workflows/build.yml

```diff
@@ -53,9 +53,14 @@ jobs:
 
       - name: Build with Pyodide
         run: |
-          set -vxeuo pipefail
+          set -vxeo pipefail
+
           git clone --depth 1 --branch ${{ matrix.version }} ${{ matrix.repo }}
           cd ${{ matrix.name }}
+
+          # Source the variables from the CF build script: https://github.com/conda-forge/pytorch-cpu-feedstock/blob/main/recipe/build.sh
+          source ../scripts/build-pytorch.sh
```

Comment:
Maybe use as another parameter - `{{ matrix.script }}`

Nit - `build-pytorch.sh` to `pytorch-build.sh if we end up adding other packages in the future